client/web/src/org/settings/DeleteOrg.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useCallback, useState } from 'react'
+import { RouteComponentProps } from 'react-router-dom'
+
+import { Button, Container } from '@sourcegraph/wildcard'
+
+import { OrgAreaPageProps } from '../area/OrgArea'
+
+import { DeleteOrgModal } from './DeleteOrgModal'
+
+interface DeleteOrgProps extends OrgAreaPageProps, RouteComponentProps<{}> {}
+
+/**
+ * Deletes an organization.
+ */
+export const DeleteOrg: React.FunctionComponent<DeleteOrgProps> = props => {
+
+    const [showDeleteModal, setShowDeleteModal] = useState(false)
+    const toggleDeleteModal = useCallback(() => setShowDeleteModal(show => !show), [setShowDeleteModal])
```

Comment:
What is the point of `show => !show` here? Isn't this enough?
```suggestion
    const toggleDeleteModal = useCallback(() => setShowDeleteModal(!showDeleteModal), [setShowDeleteModal, showDeleteModal])
```